### PR TITLE
Mod 3

### PR DESCRIPTION
**Pull request by dawnmiceli100**
This pull request includes the addition of the user reviews feature. The TDD tests for users, sessions, videos and reviews were also added.

### CODE REVIEW

File: app/models/review.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class Review < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :video
+
+  validates_presence_of :rating, :body
+  #validates :rating, inclusion: { in: (1.0..5.0), message: "rating must be from 1 to 5" }
```
Comment (alagram) [inline app/models/review.rb]: This is a tiny nitpick... unless you plan on using this in the future you can delete it.

File: app/models/video.rb [hunk 2]
```diff
@@ -10,4 +12,16 @@ def self.search_by_title(search_string)
       where("LOWER(title) LIKE ?", "%#{search_string.downcase}%").order('created_at DESC')
     end  
   end  
+
+  def average_rating
+    if self.reviews.exists?
```
Comment (alagram) [inline app/models/video.rb:18]: you can also do `reviews.average(:rating).round(1) if reviews.average(:rating)`. See [here](http://guides.rubyonrails.org/active_record_querying.html#average)

File: app/models/review.rb (hunk 1 above)
Comment (dawnmiceli100) [inline app/models/review.rb]: Thanks. I realized I won't need that since the user is selecting the values.

File: app/models/video.rb (hunk 2 above)
Comment (dawnmiceli100) [inline app/models/video.rb:18]: Thanks. That is a much more concise way to do the calculation.

**Pull request merged**